pages/facts.html

```diff
@@ -4,9 +4,46 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
+  <link href="../styles/facts.css" rel="stylesheet">
+  <title>Fun Facts | All About Coffee </title>
 </head>
 <body>
-  
+  <header>
+    <nav>
+      <ul class="site-menu">
+        <li><button onclick="window.location.href='../index.html'">HOME</button></li>
+        <li><button onclick="window.location.href='gallery.html'">GALLERY</button></li>
+        <li><button onclick="window.location.href='#'">FUN FACTS</button></li>
```

Comment:
There's an error when validated: `The element button must not appear as a descendant of the a element.`